doctor_register.php

```diff
@@ -1,31 +1,35 @@
-<?php include "header.php"; ?>
+ <?php
+  include "header.php";
+ ?>
 
-<body>
 
-  <form action="#" method="post">
+<body style="font-size: 10px;">
+
+  <form  method="post" action="register_user/register.php">
     <h2>Doctor Register</h2>
     <p>
-      <label for="Email" class="floatLabel">Doctor Id</label>
-      <input id="number" name="id" type="text">
+      <label for="username" class="floatLabel">Patient Name</label>
```

Comment:
This seems completely wrong. Are you sure you are developing a doctor register form here ? The page name says doctor_register.php and here you have Patient name. Just doesn't make sense.